src/js/components/agencyV2/overview/BarChart.jsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * BarChart.jsx
+ * Created by Max Kendall 4/13/21
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const BarChart = ({
+    agencyBudgetByYear,
+    selectedFy
+}) => {
+    const renderBars = () => {
+        const highestYear = agencyBudgetByYear.reduce((acc, obj) => {
+            if (obj.budget > acc) return obj.budget;
+            return acc;
+        }, 0);
+        return agencyBudgetByYear
+            .filter((obj) => {
+                // show preceding 4 years only.
+                if (parseInt(selectedFy, 10) <= 2021 && obj.year <= 2021) return true;
+                if (parseInt(selectedFy, 10) > 2021 && obj.year >= parseInt(selectedFy, 10) - 4) return true;
+                return false;
+            })
+            .sort((a, b) => a.year - b.year)
+            .map(({ year: fy, budget: total }, i) => {
```

Comment:
Any reason we are destructuring these variables to `fy` and `total`? 

Below when we create the height in the style property for the span `height: `${(total / highestYear) * 100}%``, It seems like `budget / greatestBudget` would make more sense?

Reply:
sure.